Update spec (#54)
NethermindEth/el-node-spec: docs/docs/introduction.md

# Introduction

> The [EigenLayer](https://www.eigenlayer.xyz/) ecosystem allows, among other incredible features, Ethereum validators to restake their ETH and provide services to the network, such as a Data Availability Layer with a particular type of node software called actively validated services (AVS). 

Reviewer: What other incredible features? If we do not specify any, I think we can remove this sentence.

```suggestion
The [EigenLayer](https://www.eigenlayer.xyz/) ecosystem allows Ethereum validators to restake their ETH and provide services to the network, such as a Data Availability Layer with a particular type of node software called actively validated services (AVS). 
```

Author: Done [b781a58](https://github.com/NethermindEth/el-node-spec/pull/54/commits/b781a58a87e5e3d868124c43497aa0122840fee6)

> With the increasing number of AVS use cases, there is a need for a convention on how to develop, maintain, create interoperability and use them. Eigenlayer has partnered with Nethermind to create an AVS Node Specification with those goals in mind. We also decided to build an AVS Node Setup Wizard to install and manage AVS Nodes that follow such specification. This documentation describes the AVS Node Specification and the AVS Node Setup Wizard.

Reviewer: Should we mention _monitoring_ in the first sentence?

Author: Done [b781a58](https://github.com/NethermindEth/el-node-spec/pull/54/commits/b781a58a87e5e3d868124c43497aa0122840fee6)

The documentation is divided into the following sections:

- [AVS Node Specification](/docs/category/avs-node-specification): A full description of the specification that covers from AVS packaging to Standard Metrics.

> - [Setup wizard](/docs/category/avs-setup-wizard): An AVS wizard setup tool, aimed at making the operations of a node operator easy.

Reviewer: ```suggestion
- [Setup wizard](/docs/category/avs-setup-wizard): An AVS wizard setup tool, aimed to simplify AVS Node management for node operators.
```

Author: Done [b781a58](https://github.com/NethermindEth/el-node-spec/pull/54/commits/b781a58a87e5e3d868124c43497aa0122840fee6)
